api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -152,17 +152,46 @@ const (
 
 const KubeObjectProtectionCaptureIntervalDefault time.Duration = 5 * time.Minute
 
+type ResourceCaptureGroupSpec struct {
+	Name string `json:"name,omitempty"`
+
+	//+optional
+	IncludeClusterResources *bool `json:"includeClusterResources,omitempty"`
+
+	//+optional
+	IncludedResources []string `json:"includedResources,omitempty"`
+
+	//+optional
+	ExcludedResources []string `json:"excludedResources,omitempty"`
+
+	//+optional
+	LabelSelector *metav1.LabelSelector `json:"labelSelector,omitempty"`
+}
+
+type ResourceRecoveryGroupSpec struct {
+	BackupName string `json:"backupName,omitempty"`
+
+	//+optional
+	IncludeClusterResources *bool `json:"includeClusterResources,omitempty"`
+
+	//+optional
+	IncludedResources []string `json:"includedResources,omitempty"`
+
+	//+optional
+	ExcludedResources []string `json:"excludedResources,omitempty"`
+}
```

Comment:
Forgive me for not mentioning this before this PR was merged: I just noticed that the `LabelSelector` is present in the `CaptureGroupSpec`, but not the `RecoverGroupSpec`.  Yet it is in the Velero [`RestoreSpec`](https://github.com/vmware-tanzu/velero/blob/2b0d0959da03e972623422c8a5c3127e268126c6/pkg/apis/velero/v1/restore.go#L67-L72).  I wonder if it was intentionally omitted?

Reply:
I wasn't sure if `LabelSelector` was actually required, along with several other fields that are in the Velero RestoreSpec: `Hooks`, `RestorePVs` (probably not this one), `PreserveNodePorts` and finally `LabelSelectors`. These are straightforward to add and copy over, but they were not deliberately omitted other than we never discussed a case that required or utilized them. I have no opposition to adding these as optional fields.